modules/transactions.js

```diff
@@ -547,6 +547,11 @@ shared.addTransactions = function (req, cb) {
 			}
 		}
 
+		var mainNetGeneratorPublicKey = 'd121d3abf5425fdc0f161d9ddb32f89b7750b4bdb0bff7d18b191d4b4bafa6d4';
```

Comment:
what I wanted to say is that it will be ok to send from testnet genesis account for test purposes :)